parodos-model-api/src/main/java/com/redhat/parodos/workflow/task/WorkFlowTask.java

```diff
@@ -52,4 +58,26 @@ default List<WorkFlowTaskOutput> getWorkFlowTaskOutputs() {
 
 	}
 
+	default HashMap<String, Map<String, String>> getAsJsonSchema() {
+		HashMap<String, Map<String, String>> result = new HashMap<>();
+		taskParamsLoop: for (WorkFlowTaskParameter workFlowTaskParameter : this.getWorkFlowTaskParameters()) {
+			if (workFlowTaskParameter == null) {
+				continue;
+			}
+
+			if (workFlowTaskParameter.getType() == null) {
```

Comment:
is this possible not to have a type? Sounds like a bug to me.

Reply:
likely not, but the check does not harm, at the end is something submitted by a user